packages/clay-card/stories/index.tsx

```diff
@@ -18,451 +21,223 @@ const portletImage = require('./static/portlet.svg');
 
 const spritemap = require('@clayui/css/lib/images/icons/icons.svg');
 
-const ClayCheckboxWithState = (props: any) => {
-	const [value, setValue] = React.useState<boolean>(false);
+const ClayCardWithImageAndState = (props: any) => {
+	const [selected, setSelected] = React.useState<boolean>(false);
 
 	return (
-		<ClayForm.Checkbox
-			checked={value}
-			disabled={false}
-			indeterminate={false}
-			onChange={() => setValue(val => !val)}
-		>
-			{props.children}
-		</ClayForm.Checkbox>
+		<ClayCardWithImage
+			{...props}
+			onSelectChange={setSelected}
+			selected={selected}
+		/>
+	);
+};
+
+const ClayCardWithuserAndState = (props: any) => {
+	const [selected, setSelected] = React.useState<boolean>(false);
+
+	return (
+		<ClayCardWithUser
+			{...props}
+			onSelectChange={setSelected}
+			selected={selected}
+		/>
+	);
+};
+const ClayCardWithFileAndState = (props: any) => {
+	const [selected, setSelected] = React.useState<boolean>(false);
+
+	return (
+		<ClayCardWithFile
+			{...props}
+			onSelectChange={setSelected}
+			selected={selected}
+		/>
+	);
+};
+const ClayCardWithFolderAndState = (props: any) => {
+	const [selected, setSelected] = React.useState<boolean>(false);
+
+	return (
+		<ClayCardWithFolder
+			{...props}
+			onSelectChange={setSelected}
+			selected={selected}
+		/>
 	);
 };
 
 storiesOf('ClayCard', module)
-	.add('with folder card', () => (
```

Comment:
Not sure its super necessary to showcase low-level in our stories since the high-level components are essentially the showcase of how we use the low-level. We can add a few low-level if you think its important though.

Reply:
Yes, I agree, but the real idea would be to show the possibilities that can be made beyond high-levels. It is not necessary to show the same thing as a high-level.